src/ffi.ts

```diff
@@ -16,27 +16,50 @@ export function unload() {
     lib.close();
 }
 
-let lib_file;
+const lib_file = (await getLibFile()).default;
```

Comment:
This is necessarily a resolved Promise to facilitate the  [issue given below](https://github.com/tr1ckydev/webview-bun/pull/15/files#r1810943964).

Reply:
Is `bun build` able to dead code eliminate when wrapped inside a function?

Reply:
I had this set up in a sandbox at the time of the PR, and tested it manually. It all worked. You will have to test it alongside your new build script. I suggest that you start writing some unit and functional tests for your project, otherwise it will quickly become impossible for other people to contribute effectively.